AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/DefaultAudioVideoControllerTests.swift

```diff
@@ -173,7 +182,28 @@ class DefaultAudioVideoControllerTests: CommonTestCase {
             callKitEnabled: false,
             audioMode: .stereo48K,
             audioDeviceCapabilities: .inputAndOutput,
-            enableAudioRedundancy: false
+            enableAudioRedundancy: false,
+            reconnectTimeoutMs: self.reconnectTimeoutMs
+        )).wasCalled()
+        verify(videoClientControllerMock.start()).wasCalled()
+    }
+    
+    func testStart_120000ReconnectTimeoutMs() {
+        let testReconnectTimeoutMs = 120 * 1000
```

Comment:
I have a question about why test 120s while reconnect timeout is 180

Reply:
> I have a question about why test 120s while reconnect timeout is 180

180 is default value, for testing purpose we can to make sure other values also work.